polkadot/runtime/parachains/src/inclusion/mod.rs

```diff
@@ -1098,40 +1100,52 @@ impl<T: Config> Pallet<T> {
 	/// This should generally not be used but it is useful during execution of Runtime APIs,
 	/// where the changes to the state are expected to be discarded directly after.
 	pub(crate) fn force_enact(para: ParaId) {
-		let pending = <PendingAvailability<T>>::take(&para);
-		let commitments = <PendingAvailabilityCommitments<T>>::take(&para);
-
-		if let (Some(pending), Some(commitments)) = (pending, commitments) {
-			let candidate =
-				CommittedCandidateReceipt { descriptor: pending.descriptor, commitments };
+		// This does not take elastic-scaling into account, it enacts the first candidate.
+		let enacted_candidate =
+			<PendingAvailability<T>>::mutate(&para, |candidates| match candidates {
+				Some(candidates) => candidates.pop_front(),
+				// TODO: this should also check the descendants, as they have been made available
+				// before their parent. Or just change the semantic of force_enact to enact all
+				// candidates of a para.
+				_ => None,
```

Comment:
is there an issue for this TODO?

Reply:
this should be tackled in this PR. I'm curious what you and other think about it. The easiest way would be to have force_enact enact all pending candidates of the para